_config.yml

```diff
@@ -2,7 +2,6 @@
 #      Main Configs       #
 # ----------------------- #
 
-baseurl: /mentoring
```

Comment:
Are you sure we don't need this? Because I had some problems with broken links without it ... but it was in my user space ...